src/js/components/TodoList.js

```diff
@@ -0,0 +1,97 @@
+import { $, $$ } from "../utils/querySelector.js";
+import API from "../api/api.js";
+
+export default function TodoList ({ reloadTodos, filterTodos }) {
+	this.userId = "";
+
+	const $new = ({ _id, contents, isCompleted, priority }) => {
+		return `
+			<li class="${ isCompleted && "completed" }" data-id="${ _id }">
+				<div class="view">
+					<input class="toggle" type="checkbox" data-id="${ _id }" ${ isCompleted && "checked" }/>
+					<label class="label">
+						${ setPriority(priority) }
+						${ contents }
+					</label>
+					<button class="destroy"></button>
+				</div>
+				<input class="edit" value="${ contents }" />
+			</li>`;
+	};
+
+	const setPriority = (priority) => {
+		let item = "";
+		if (priority === "NONE")
+			item = `<select class="chip select">
+						<option value="0" selected>순위</option>
+						<option value="1">1순위</option>
+						<option value="2">2순위</option>
+					</select>`
+		else if (priority === "FIRST") item = `<span class="chip primary">1순위</span>`
+		else if (priority === "SECOND") item = `<span class="chip secondary">2순위</span>`
+
+		return item;
+	}
+
+	const editItem = ({ currentTarget }) => {
+		const $parentLi = currentTarget.parentElement.parentElement;
```

Comment:
parentElement.parentElement로 접근할 수도 있지만 closest()라는 메서드를 통해서 가장 가까운 태그를 찾을 수도 있더라구요!

```suggestion
		const $parentLi = currentTarget.closest('li);
```